LMS/bookinventery/models.py

```diff
@@ -22,17 +24,31 @@ def __str__(self):
         return self.title
 
 class Transaction(models.Model):
+    ''' Books Transaction who issue book,which book are issue,how many time he/she ready the book '''
+
     STATUS = ((0,'Pending'),(1,'Issue'),(2,'Return'))
```

Comment:
STATUS should be moved to the constants file. As we need to use this in the views file.

Reply:
> file

Thanks for review. That's why we will understanding better code